tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -295,6 +295,12 @@ ROOT::Experimental::RNTupleWriter::~RNTupleWriter()
       CommitCluster(true /* commitClusterGroup */);
       fSink->CommitDataset();
    } catch (const RException &err) {
+#ifdef R__USE_IMT
```

Comment:
What would happen if this code were exposed also for the `!R__USE_IMT` case? Isn't `fZipTasks->Wait();` just a no-op then?

Reply:
Actually, in that case `fZipTasks` should be `nullptr`, and thus can be safely exposed for `!R__USE_IMT`.